test/acceptance/display_matching_invocations_alongside_expectations_test.rb

```diff
@@ -24,16 +24,12 @@ def test_should_display_results
       foo.bar(1, 2) { |_ignored| }
       assert_raise(StandardError) { foo.bar(3, 4) { |_ignored| } }
     end
-    assert_failed(test_result)
-    assert_equal [
-      'not all expectations were satisfied',
-      'unsatisfied expectations:',
-      '- expected exactly once, not yet invoked: #<Mock:foo>.bar(1)',
-      'satisfied expectations:',
+    assert_invocations(
```

Comment:
Minor: It would be helpful to expand the commit note ("DRY up tests") a bit to explain which tests and possibly mention extracting the `assert_invocations` method.